ext/spl/tests/gh10907.phpt

```diff
@@ -0,0 +1,135 @@
+--TEST--
+GH-10907 (Unable to serialize processed SplFixedArrays in PHP 8.2.4)
+--FILE--
+<?php
+// Test without rebuilding properties
```

Comment:
Could you ``echo`` the comments? It makes understanding what var_dumps correspond to what easier IMHO

Reply:
Thx for the review, makes sense. I've done that now (ofc I'll squash that into the first commit when I merge this)